packages/alfa-aria/src/feature.ts

```diff
@@ -495,7 +458,47 @@ const Features: Features = {
           yield Attribute.of("aria-placeholder", value);
         }
       },
-      nameFromLabel
+      (element, device, state) => {
+        if (
+          test<Element>(
+            hasInputType("text", "password", "search", "tel", "url"),
+            element
+          )
+        ) {
+          return Name.fromSteps(
+            () => nameFromLabel(element, device, state),
+            () => nameFromAttribute(element, "title"),
+            () => nameFromAttribute(element, "placeholder")
```

Comment:
I'm considering whether `nameFromAttribute` should accept several attribute names and test them in order: `nameFromAttribute(element, "title", "placeholder")`.

Reply:
Seems like a fine idea 👍